### ISSUE TITLE
90 minute Sandbox Getting started guide

### ISSUE THREAD

**Issue by critesjosh** [AztecProtocol/dev-rel#40]
- Actually write code (JS and Noir)
- Javascript [guide](https://aztec-docs-dev.netlify.app/dev_docs/getting_started/sandbox)
- Go over the Development flow
    - Edit a contract, recompile, redeploy, test. Repeat. 
    - https://aztec-docs-dev.netlify.app/dev_docs/contracts/compiling
- Frontend not needed (optional)
- “Unbox” zktoken, then go through the contract development flow (Dan Lee may be working on this) 
- Deliverable: guide on how to iteratively develop an Aztec contract

**Comment by catmcgee** [comment]
working on this

**Comment by iAmMichaelConnor** [comment]
> “Unbox” zktoken, then go through the contract development flow (Dan Lee may be working on this)

Yep - Dan & Leila are working on an `aztec-cli unbox ...` command, which unboxes the PrivateTokenContract. Please do chat with them :)

**Comment by critesjosh** [comment]
WIP [basic example contract here](https://github.com/critesjosh/example-private-token/tree/main) with basic instructions in README and typescript testing set up. I started this repo with the example contract that alvaro discussed in his sandbox video. I also added a private storage variable to track the `owner` of the contract as well as `mint` and `transfer_ownership` functions that use the `owner` storage var.
